### ISSUE TITLE
How to connect to OpenDaylight using mininet 2.2.2?

### ISSUE THREAD

**Issue by Dhiraj240** [mininet/mininet#949]
I used ```sudo mn --controller=remote,ip=127.0.0.1 --mac --topo=linear,3 --switch ovsk,protocols=OpenFlow13```
which opens up a verbose to use a set of commands, is there any change in the newer mininet version command written above which were supported in older versions.

Check this screenshot
![Screenshot (419)](https://user-images.githubusercontent.com/29758643/84468976-a10c4400-ac9d-11ea-94b0-e0cd4cde7f3e.png)

**Comment by Dhiraj240** [comment]
Please configure it with the newer mininet 2.2.2 and ODL magnesium so that I can update details.

**Comment by Giuseppe1992** [comment]
The command looks correct
`sudo mn --controller=remote,ip=127.0.0.1 --mac --topo=linear,3 --switch ovsk,protocols=OpenFlow13`

But in the screenshot, it seems that you have a space between `--controller=remote, ip...` and `--topo=linear, 3`

**Comment by Dhiraj240** [comment]
Yes working now, but I can't see any flows and get output from rest api in postman now using ODL as controller.
